concent_api/core/tests/test_unit_model.py

```diff
@@ -336,3 +343,91 @@ def test_that_deposit_account_is_not_removed_when_deposit_claim_is_deleted(self)
     def test_that_no_exception_is_raised_when_deposit_claim_is_valid(self):
         self.deposit_claim.clean()
         self.deposit_claim.save()
+
+
+class CoreViewReceiveTest(ConcentIntegrationTestCase):
```

Comment:
`CoreViewReceiveTest`? Wrong name. Why didn't you just add it to `TestSubtaskModelValidation` tests?

Reply:
I cannot move it because it has to inherit from  `ConcentIntegrationTestCase`. Renamed to `ProtocolVersionValidationTest`